### DISCUSSION TITLE
Adding Aspect ratios

### DISCUSSION THREAD

**Discussion by VanithaVijayakumar** [TimOliver/TOCropViewController#499] [General]
Could you please add a few more aspect ratios in the image cropper?

**Comment by techjavier** [comment]
Hi, I think you can set the cropViewController.customAspectRatio for a custom aspect ratio
<img width="1098" alt="Screen Shot 2022-01-06 at 22 39 04" src="https://user-images.githubusercontent.com/53168317/148399685-f9381968-32c8-43fa-99d8-6a15058380c6.png">
However, I could not figure how to set it 
This was in the documentation
/**
 A CGSize value representing a custom aspect ratio, not listed in the presets.
 E.g. A ratio of 4:3 would be represented as (CGSize){4.0f, 3.0f}
 */

**Comment by VanithaVijayakumar** [comment]
Hi TimOliver,
 Thank you for your reply I was able to add the custom aspect ratios.

**Comment by techjavier** [comment]
Turns out the reason it wasn't working it was because the example he gave was an objective c syntax.
This is the answer I got when I asked him on twitter:

Hey Javier! It's supposed to match the same pattern as normal aspect ratios.

So to make a 16:9 ratio like a standard PC monitor, you would set it as:

CGSize(width: 16.0, height: 9.0)

Good luck!
Also, the main reason why this isn't working here is that CGSize syntax is Objective-C, but I can see you're working in Swift. I provided the equivalent Swift syntax above! 

So the swift syntax equivalent would be this CGSize(width: 16.0, height: 9.0)

**Comment by VanithaVijayakumar** [comment]
Thank you for the clarification.
